os/arch/arm/src/amebasmart/amebasmart_i2s.c

```diff
@@ -122,6 +122,10 @@
 #define I2S_DMA_PAGE_SIZE 4096 	/* 4 ~ 16384, set to a factor of APB size */
 #define I2S_DMA_PAGE_NUM 4	/* Vaild number is 2~4 */
 
+#ifdef CONFIG_PM
+static bool i2s_lock_state = 0;
```

Comment:
static volatile

Reply:
@zhongnuo-tang Could you check @edwakuwaku 's comment?